ui/src/data/DataStoreBase.ts

```diff
@@ -105,6 +127,7 @@ export class DataStoreBase {
     };
     return async () => {
       await this.zomeApi.put_thing(itemThing);
+      await this.zomeApi.signal_call('path');
```

Comment:
Is this correct, shouldn't it be getting the actual path and not the string 'path'?

Reply:
Yeah for sure. Depending on how the state stuff is implemented, this may change so I just left it as such for now.